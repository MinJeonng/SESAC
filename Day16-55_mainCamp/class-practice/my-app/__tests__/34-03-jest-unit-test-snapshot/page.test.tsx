import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import JestUnitTestPage from '@/app/section34/34-03-jest-unit-test-snapshot/page';

it('기존 사진이랑 바뀐게 없는지 비교해보자 with snapshotTest', () => {
  const result = render(<JestUnitTestPage />);
  //기존 사진이랑 비교 이걸 실행하면 새로운 snapshot 폴더가 생겨서 사진 찍힌 것처럼 저장됌
  //34-02 파일처럼 일일히 다 쓰는게 아니라 스냅샷을 찍으면 훨씬 간딘힘
  expect(result.container).toMatchSnapshot();
});
