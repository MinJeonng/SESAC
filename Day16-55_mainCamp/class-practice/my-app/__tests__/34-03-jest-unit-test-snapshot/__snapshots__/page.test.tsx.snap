// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`기존 사진이랑 바뀐게 없는지 비교해보자 with snapshotTest 1`] = `
<div>
  <div>
    jestUI테스트 중입니다.
  </div>
  예시 1 : 
  <input
    type="text"
  />
  <button>
    예시 1
  </button>
</div>
`;
